src/main/webapp/src/App-clear.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import { MDBBtn, MDBCol, MDBContainer, MDBRow } from "mdbreact";
+import "./index.css";
+import logo from "./logo.png";
+
+class App extends Component {
+  render() {
+    return (
+      <MDBContainer>
+        <MDBRow center style={{ height: "100vh" }}>
+          <MDBCol middle="true" sm="8" className="text-center">
+            <img src={logo} alt="logo" style={{ width: "10rem" }} />
+            <h1>Welcome to Your MDB React App</h1>

```

Comment:
Can we change this to "weTime"?

Reply:
Yes, I'll add it in the next pull request.